transports/msmq/distributor/disconnect-workers.md

```diff
@@ -10,6 +10,10 @@ redirects:
 
 The Distributor starts sending messages to a Worker once it is aware of it. A Worker registers itself with a Distributor by sending a message containing a `SessionID` that identifies the current running Worker and the number of messages it can handle concurrently.
 
+## Prerequisites
+
+Prior to installation ensure that PowerShell 2 or higher is installed. NServiceBus PowerShell modules are compatible with PowerShell 5. Versions of PowerShell later than 5 are not supported and might not work as expected.
```

Comment:
Again, do we need to call out PSC?

```suggestion
Prior to installation ensure that PowerShell 2 or higher is installed. NServiceBus PowerShell modules are compatible with PowerShell 5. Versions of PowerShell later than 5 (including PowerShell Core) are not supported and might not work as expected.
```

Reply:
It doesn't hurt. I'll add it everywhere.